### DISCUSSION TITLE
Applicative Functor

### DISCUSSION THREAD

**Discussion by FizzyElt** [FizzyElt/functional-programming#80] [General]
接續 Functor 的方式講述範疇論跟程式之間的關聯

參考資源
- https://lean-lang.org/functional_programming_in_lean/functor-applicative-monad.html
- https://ncatlab.org/nlab/show/monoidal+functor

**Comment by dannypsnl** [comment]
https://www.cambridge.org/core/journals/journal-of-functional-programming/article/applicative-programming-with-effects/C80616ACD5687ABDC86D2B341E83D298

**Comment by dannypsnl** [comment]
我的建議順序是
1. F-algebra
2. Monad
3. Applicative 

因為第一個是前置條件，而 monad 比 applicative 容易的多

**Reply by FizzyElt** [reply]
我順便讀一下 F-algebra

**Comment by dannypsnl** [comment]
https://mathstodon.xyz/@MartinEscardo/111506730655734081

考慮這個，應該把 applicative 當成補充而不是主題

**Comment by FizzyElt** [comment]
那這個先關掉
